CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.17.0]
+- **New**
+    - Add a new pipeline build_pipeline_repetead_learners that supports multiple learners with the same name.
+
```

Comment:
You can remove this one as we update the changelog in the release PR with all the changes.

Reply:
👍